Projects/My_website/myWebsite/myWebsite/settings.py

```diff
@@ -85,6 +85,8 @@
         'PASSWORD': 'ummed',
         'HOST': 'localhost',
         'PORT': '5432'
+        # 'ENGINE': 'django.db.backends.sqlite3',
+        # 'NAME': BASE_DIR / 'db.sqlite3',
```

Comment:
@codal-uchoudhary , no need to keep this commented if not needed at all.  Remove it.